BeeKit/Model/Goal.swift

```diff
@@ -62,6 +62,14 @@ public class Goal: NSManagedObject {
     @NSManaged public func addToRecentData(_ values: Set<DataPoint>)
     @objc(removeRecentData:)
     @NSManaged public func removeFromRecentData(_ values: Set<DataPoint>)
+    
+    @NSManaged public var dueByTableData: String?
+    
+    public var dueBy: [String: BeeminderDueByEntry] {
+        self.loadDueByTable() ?? [:]
+    }
```

Comment:
Can we not use a [ValueTransformer](UICollectionViewDiffableDataSource) to avoid maintaining both properties?

Reply:
I think you meant to link to something else with the ValueTransformer link there. I wanted to avoid making a separate core data entity for the due by table. I can look into the value transformer.

Reply:
You're right, I was trying to link to https://developer.apple.com/documentation/coredata/handling_different_data_types_in_core_data#3582174

Although https://developer.apple.com/documentation/foundation/valuetransformer might have been a better link

Reply:
done